spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -92,6 +92,71 @@
       end
     end
 
+    describe "exemptions" do
+      context "single filer" do
+        let(:submission) {
+          create :efile_submission, tax_return: nil, data_source: create(
+            :state_file_nj_intake,
+            primary_first_name: "Bert",
+            primary_last_name: "Muppet",
+            primary_middle_initial: "S",
+          ) }
+        it "fills pdf with correct Line 6 fields" do
+          expect(pdf_fields["Check Box39"]).to eq "Off"
+          expect(pdf_fields["Check Box40"]).to eq "Off"
+          expect(pdf_fields["Domestic"]).to eq "1"
+          expect(pdf_fields["x  1000"]).to eq "1000"
+        end
+
+        it "fills pdf with Line 7 fields" do
+          expect(pdf_fields["Check Box41"]).to eq "Off"
+          expect(pdf_fields["Check Box42"]).to eq "Off"
+          expect(pdf_fields["x  1000_2"]).to eq "0"
+          expect(pdf_fields["undefined_9"]).to eq "0"
+        end
+        context "primary over 65" do
```

Comment:
[nitpick] I prefer a line of whitespace in between context blocks / describe blocks / it blocks

Reply:
ran rubocop, thanks for reminder